crates/revm/Cargo.toml

```diff
@@ -15,11 +15,13 @@ bytes = { version = "1.1", default-features = false }
 #needed for web3 db connector
 futures = { version = "0.3.21", optional = true }
 hashbrown = { version = "0.12" }
+hex = { version = "0.4.3", optional = true }
```

Comment:
remove last number, remove hex from `dev-dependencies`, and do a build so that cargo.lock is updated.

Reply:
was already locked at 0.4.3